src/components/material/periodical/MaterialPeriodical.tsx

```diff
@@ -22,7 +22,8 @@ const MaterialPeriodical: FC<MaterialPeriodicalProps> = ({
 
   const { data, isLoading, isError } = useGetHoldings({
     faustIds: [faustId],
-    config
+    config,
+    blacklist: "pickup"
```

Comment:
:+1: This previously used the default value `false` for `useAvailabilityBlacklist`. This was mapped to the `pickup` type. Now it is the `pickup` type. Behavior should be unchanged.